README.md

```diff
@@ -1,194 +1,115 @@
-        objectbox_flutter_libs: any
-      ```
-    * Install the packages `flutter pub get`
-    * XCode/iOS: under Architectures replace `${ARCHS_STANDARD)` with `arm64` (or `$ARCHS_STANDARD_64_BIT`).
-      See [FAQ](#faq) for details.
-1. **Dart standalone programs**:
-    * Install the packages `pub get`
-    * Install [objectbox-c](https://github.com/objectbox/objectbox-c) system-wide:
-       * macOS/Linux: execute the following command (answer Y when it asks about installing to /usr/lib) 
-            ```shell script
-            bash <(curl -s https://raw.githubusercontent.com/objectbox/objectbox-dart/main/install.sh)
-            ```
-       * macOS: if dart later complains that it cannot find the `libobjectbox.dylib` you probably have to unsign the 
-         `dart` binary (source: [dart issue](https://github.com/dart-lang/sdk/issues/38314#issuecomment-534102841)):
-            ```shell script
-            sudo codesign --remove-signature $(which dart)
-            ```
-       * Windows: use "Git Bash" or similar to execute the following command 
-            ```shell script
-            bash <(curl -s https://raw.githubusercontent.com/objectbox/objectbox-dart/main/install.sh)
-            ```
-            Then copy the downloaded `lib/objectbox.dll` to `C:\Windows\System32\` (requires admin privileges).
-
-ObjectBox generates code binding code for classes you want stored based using build_runner.
-After you've defined your persisted entities (see below), run `pub run build_runner build` or `flutter pub run build_runner build`.
-
-Getting started
-----------------
-In general, Dart class annotations are used to mark classes as ObjectBox entities and provide meta information.
-Entity IDs and UIDs that are defined in their respective annotations need to be unique across all entities, while 
-property IDs only need to be unique in their respective entity; property UIDs also need to be globally unique.
+![ObjectBox logo](https://raw.githubusercontent.com/objectbox/objectbox-java/master/logo.png)
 
-Each entity is required to have an ID property of type `int`.
-Already persisted entities have an ID greater or equal to 1.
-New (not yet persisted) objects typically have ID value of `0` or `null`: calling `Box.put` automatically assigns a new ID to the object.
+ObjectBox Dart & Flutter
+==========================
 
-### Example
-For a code example, see [example/README.md](example/README.md)
+ObjectBox is a super-fast database storing Dart objects locally.
 
-### Box
-Box is your main interface for storing and retrieving data.
-```dart
-var box = Box<Note>(store);
-    
-var note = Note(text: "Hello");
-note.id = box.put(note);
-print("new note got id ${note.id}");
-print("refetched note: ${box.get(note.id)}");
-```
+* 🏁 **High performance** - improving response rates and enabling real-time applications.
+* 🪂 **ACID compliance** - Atomic, Consistent, Isolated, Durable.
+* 🔗 **Relations** - object links / relationships are build-in.
```

Comment:
```suggestion
* 🔗 **Relations** - object links / relationships are built-in.
```

Reply:
of course you're right... copy paste from https://objectbox.io/dev-get-started/